implementations/monix/src/test/scala/com/softwaremill/sttp/impl/monix/package.scala

```diff
@@ -0,0 +1,15 @@
+package com.softwaremill.sttp.impl
+
+import scala.concurrent.Future
+
+import _root_.monix.eval.Task
```

Comment:
also minor, we're not using `_root_` anywhere so I would stick to that convention

Reply:
It is needed in this case as the package object is called monix. I have removed some other unnecessary ones however.

Reply:
makes sense, missed the package name conlict